### PR TITLE
decred: Return address with no peers.

### PR DESCRIPTION
**Pull request by JoeGruffins**
# Description

Allows users to get an address if we were ever synced and have just lost peers.

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: cw_decred/lib/wallet_addresses.dart [hunk 1]
```diff
@@ -7,13 +7,27 @@ import 'package:cw_decred/api/libdcrwallet.dart' as libdcrwallet;
...
+    final nAddr = libdcrwallet.newExternalAddress(walletInfo.name) ?? '';
+    if (nAddr != '') {
+      currentAddr = nAddr;
+    }
+    return nAddr;
+  }
+
+  String get currentAddress {
+    return currentAddr;
   }
```
Comment (ukane-philemon) [inline cw_decred/lib/wallet_addresses.dart]: Is this getter used outside decred lib?

Comment (JoeGruffins) [reply cw_decred/lib/wallet_addresses.dart]: No. Was there a better way of doing this? Can I just access the fields?

Comment (JoeGruffins) [reply cw_decred/lib/wallet_addresses.dart]: Yeah looks like I can so changed it https://github.com/JoeGruffins/cake_wallet/compare/bd6be939d67b6733abce0f9dbc8a72dbc5e99d9e..bf1900c96c46e2ebb44b1358daf7d98661053171

**Review by ukane-philemon — APPROVED** [review]
utAck.

**Comment by JoeGruffins** [comment]
Thanks for review. Added to the main pr.

**Pull request closed without merging**